src/test/java/com/wikia/webdriver/testcases/adstests/TestVideoVelesMobile.java

```diff
@@ -51,7 +51,7 @@ public void adsVelesWithoutOfferEvent() {
 
     velesAds.triggerIncontentPlayer();
     velesAds.wait.forSuccessfulResponseByUrlPattern(networkTrafficInterceptor, AdsVelesObject.NO_OFFER_PLAYER_EVENT_PATTERN);
-  }
+    Assertion.assertTrue(velesAds.isVideoHidden(), "Video player not hidden");}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move this closing curly brace to the next line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)